### PR TITLE
UI-Kit via SCSS import

### PR DESCRIPTION
**Pull request by maxious**
Depends on https://github.com/AusDTO/gov-au-ui-kit/pull/183

The homepage issues after including UI-Kit were caused by a change in the desktop screen width and columns from 12 to 16. Also the light grey used in the homepage prototype is no longer available.

### CODE REVIEW

File: vendor/assets/update-ui-kit.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+wget http://gov-au-ui-kit.apps.staging.digital.gov.au/latest/ui-kit.js -O javascripts/ui-kit.js
+wget http://gov-au-ui-kit.apps.staging.digital.gov.au/latest/_ui-kit.scss -O stylesheets/_ui-kit.scss
+wget http://gov-au-ui-kit.apps.staging.digital.gov.au/latest/ui-kit-icons.css -O stylesheets/ui-kit-icons.css
```
Comment (tessereth) [inline vendor/assets/update-ui-kit.sh]: Question: does it make more sense for this to be here or `/bin`? It's an executable file but it's to do with the assets...

**Comment by tessereth** [comment]
Changelog?

**Comment by tessereth** [comment]
LGTM

File: app/assets/stylesheets/_homepage.scss [hunk 2]
```diff
@@ -1,5 +1,4 @@
-$desktop: (min-width 1000px);
-
+$lighterer-grey: #f0f0f0;
```
Comment (micapam) [inline app/assets/stylesheets/_homepage.scss]: lolol

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -10,18 +10,14 @@
...
  *= require "simplemde.min"
  *= require "trumbowyg.min"
  *= require "editor.md/scss/editormd"
  */
- @import "bourbon";
- @import "neat";
+ @import "../../../vendor/assets/stylesheets/ui-kit";
+ @import "../../../vendor/assets/stylesheets/ui-kit-icons";
 
- @import "colours";
```
Comment (micapam) [inline app/assets/stylesheets/application.scss]: I see colours.scss has gone ... how will the colours work now ? Are they in UIKit anyway ?

**Comment by ltankey** [comment]
LGTM

File: app/assets/stylesheets/_homepage.scss (hunk 2 above)
Comment (maxious) [inline app/assets/stylesheets/_homepage.scss]: Not an official colour... anymore

File: app/assets/stylesheets/application.scss (hunk 3 above)
Comment (maxious) [inline app/assets/stylesheets/application.scss]: Yep, particularly for the homepage we need some grid settings and colors. By including the UIKIt this way we have access to them.

**Pull request merged**